modules/utility/preview/default.nix

```diff
@@ -0,0 +1,6 @@
+_: {
```

Comment:
you can omit the `_:` as this file is not likely to receive an argset in the future